publisher/product/ontology/build-theallfile.sh

```diff
@@ -32,7 +32,7 @@ function ontologyCreateTheAllTtlFile() {
       --dir="${INPUT}/LCC" \
       --dir=${tag_root} \
       --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/AboutFIBODev/" \
-      --top="https://spec.edmcouncil.org/fibo/ontology/MetadataFIBO/" \
+      --top="https://spec.edmcouncil.org/${ONTPUB_FAMILY}/ontology/Metadata${ONTPUB_FAMILY}/" \
```

Comment:
Should be uppercase:
`--top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/Metadata${ONTPUB_FAMILY^^}/" \`